components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/SynapseAnalyticsDataProvider.java

```diff
@@ -405,9 +405,29 @@ public Map<String, Object> getProperties() {
         customProperties.put(Constants.API_CONTEXT_KEY, getApiContext());
         customProperties.put(Constants.RESPONSE_SIZE, getResponseSize());
         customProperties.put(Constants.RESPONSE_CONTENT_TYPE, getResponseContentType());
+        if (messageContext.getProperty(APIConstants.API_TYPE) == "GRAPHQL"){
```

Comment:
Do not use hard-coded values here

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/12441/commits/226c49a261240dc5305f96d69aadb6f86edfae84